libs/librrgraph/src/io/rr_graph_uxsdcxx_serializer.h

```diff
@@ -534,7 +534,7 @@ class RrGraphSerializer final : public uxsd::RrGraphBase<RrGraphContextTypes> {
             }
         }
         if (!found_arch_name) {
-            report_error("Switch name '%s' not found in architecture\n", string_name.c_str());
+            VTR_LOG_WARN("Switch name '%s' not found in architecture\n", string_name.c_str());
```

Comment:
Probably we should not make this a warning, and instead just make it an info message saying something like:
"Switch name '%s' found in RR graph input from file but not in architecture file; creating it.\n", string_name.c_str());

Otherwise you are likely to get many warnings with the RR graphs you are bringing in, and they will be spurious.